jobs/e2e_node/image-config-serial.yaml

```diff
@@ -6,14 +6,14 @@ images:
     image: ubuntu-gke-1804-1-16-v20200330 # docker 17.03
     project: ubuntu-os-gke-cloud
   cos-stable2:
-    image_regex: cos-stable-73-11647-510-0 # docker v18.09.7, deprecated after 2020-06-19
+    image_regex: cos-73-11647-534-0 # docker v18.09.7, deprecated after 2020-06-19
```

Comment:
General comment, please update or remove the comment with the updated images.

Reply:
It doesn't need to be updated or removed. It's the same milestone, so the docker version and deprecation date is the same.